src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -830,6 +958,42 @@ def __init__(
         groovedensity=None,
         **kwargs,
     ):
+        """
+        Initialize the model.
+
+        Parameters
+        ----------
+        init : str, None, optional
+            Input file name in ASDF format from which to initialize the model.
+        angle : float, optional
+            Angle between the front surface and back surface of the prism in degrees,
+            by default None.
+        gwa_tiltx, gwa_tilty : float, optional
+            Angles in x, y between the grating surface and the reference surface (MIRROR)
+            in degrees, by default None.
+        kcoef : list, optional
+            Coefficients K1, K2, and K3 to describe the material of the prism,
+            by default None.
+        lcoef : list, optional
+            Coefficients L1, L2, and L3 to describe the material of the prism,
+            by default None.
+        tcoef : list, optional
+            Six constants (D0, D1, D2, E0, E1 and lambdak) to describe
```

Comment:
Should this be `lambda_k`?
```suggestion
            Six constants (D0, D1, D2, E0, E1 and lambda_k) to describe
```